api/requirements.txt

```diff
@@ -1,11 +1,11 @@
-Django~=2.1.5
+Django~=2.2.0
 mysqlclient~=1.4.0
 djangorestframework~=3.7.0  # djangorestframework-bulk 0.2.0 stops us from upgrading to 3.8
-djoser~=1.4.0
-dnspython~=1.15.0
+djoser~=1.5.1
+dnspython~=1.16.0
 httpretty~=0.9.0
 requests~=2.21.0
 uwsgi~=2.0.0
 django-nocaptcha-recaptcha==0.0.20  # updated manually
-sqlparse~=0.2.0
+sqlparse~=0.3.0
```

Comment:
We can drop this, see https://docs.djangoproject.com/en/2.2/releases/2.2/#sqlparse-is-required-dependency